<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>모두의 이유식</string>
  </property>
  <widget class="QPushButton" name="pushbtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>원하는 메뉴를 선택해주세요</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_w">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>311</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
